File: rxjavademo/src/main/java/com/sheenhill/rxjavademo/MainActivity.java
package com.sheenhill.rxjavademo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        Observable.create(new ObservableOnSubscribe<String>() {
            @Override
            public void subscribe(Emitter<String> emitter) {
                emitter.onNext("HHHHH");
            }
        }).subscribeObserver(new Observer() {
            @Override
            public void onNext(Object o) {
                Log.d("zzzz","msg:"+o.toString());
            }

            @Override
            public void onSubscribe() {

            }

            @Override
            public void onError(Throwable e) {

            }

            @Override
            public void onComplete() {

            }
        });
    }
}

File: module_chest/src/main/java/com/sheenhill/module_chest/MainActivity.kt
package com.sheenhill.module_chest

import android.os.Bundle
import com.sheenhill.common.activity.K_BaseActivity


//class MainActivity : BaseTempActivity(StudyPlanFragment())
class MainActivity : K_BaseActivity(R.layout.activity_main){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setLightStatusBarStyle()
    }
}
